### PR TITLE
Add go language

### PR DESCRIPTION
**Pull request by terra-naut**
add support for go language to container

### CODE REVIEW

**Comment by terra-naut** [comment]
@tbugfinder

File: terraform/docker/Dockerfile.terrarium [hunk 1]
```diff
@@ -191,6 +192,13 @@ RUN TARGETARCH_SYNONYM=$([[ "$TARGETARCH" == "amd64" ]] && echo "x86_64" || echo
     && /tmp/zoxide.sh \
     && rm -f /tmp/zoxide.sh
 
+    # Install GO
```
Comment (tbugfinder) [inline terraform/docker/Dockerfile.terrarium:195]: Shouldn't be indented...

**Review by tbugfinder — CHANGES_REQUESTED** [review]

File: terraform/docker/Dockerfile.terrarium (hunk 1 above)
Comment (terra-naut) [reply terraform/docker/Dockerfile.terrarium]: fixed

**Review by tbugfinder — APPROVED** [review]
ok

**Pull request merged**